.circleci/config.yml

```diff
@@ -65,6 +65,21 @@ jobs:
           path: ~/log.txt
           destination: log.txt
 
+  pypy3:
```

Comment:
If we're going to do it on CRON in master, do that here, and make a separate PR to 0.20.X?